Lib/test/test_statistics.py

```diff
@@ -1476,6 +1521,20 @@ def test_negative_error(self):
             with self.subTest(values=values):
                 self.assertRaises(exc, self.func, values)
 
+    def test_single_value_unsupported_type(self):
+        with self.assertRaises(TypeError):
+            self.func(['3.14'])
+
+    def test_multiple_values_type_error(self):
```

Comment:
This could be combined with the previous test method.

Reply:
Indeed, now they assert the same behavior, with the same way. I will combine them under a test with name `test_invalid_type_error`.